src/Azure/Orleans.Streaming.EventHubs/Hosting/ClientBuilderExtensions.cs

```diff
@@ -8,13 +8,24 @@ namespace Orleans.Hosting
     public static class ClientBuilderExtensions
     {
         /// <summary>
-        /// Configure cluster client to use event hub persistent streams.
+        /// Configure cluster client to use event hub persistent streams. This return a configurator which allows further configuration
         /// </summary>
-        public static ClusterClientEventHubStreamConfigurator AddEventHubStreams(
+        public static ClusterClientEventHubStreamConfigurator ConfigureEventHubStreams(
```

Comment:
This works, but I worry that it won't really be clear if one is adding a new provider or configuring an existing one.

Suggest an override of AddEventHubStreams something like:

    public static ClusterClientEventHubStreamConfigurator AddEventHubStreams( 
            this IClientBuilder builder, 
            string name) 

or

    public static IClientBuilder AddEventHubStreams( 
            this IClientBuilder builder, 
            string name,
            Action<ClusterClientEventHubStreamConfigurator> configure) 

ConfigureEventHubStreams works though, if you're not fond of the suggested patterns.   Just suggestions.

Reply:
this pattern doesn't solve the problem of telling user what config is required to configure , which is mainly why I added AddXXXStreams and ConfigureXXXStreams.

Reply:
nvm. I see your point. Will add those overrides